simple-git-hooks.js

```diff
@@ -210,9 +210,19 @@ function _setHook(hook, command, projectRoot=process.cwd()) {
 /**
  * Deletes all git hooks
  * @param {string} projectRoot
+ * @param {string[]} [argv]
  */
-function removeHooks(projectRoot=process.cwd()) {
-    for (let configEntry of VALID_GIT_HOOKS) {
+function removeHooks(projectRoot = process.cwd(), argv = process.argv) {
```

Comment:
this pattern seems a bit strange, why do you need to pass argv as argument? Seems like you do not use it in your code